paperscraper/lib.py

```diff
@@ -60,14 +61,17 @@ def clean_upbibtex(bibtex):
     return bibtex
 
 
-def format_bibtex(bibtex, key):
+def format_bibtex(bibtex, key, clean=True):
```

Comment:
```suggestion
def format_bibtex(bibtex, key, clean: bool = True) -> str:
```

Bonus points for type hinting `bibtex` as well